sdk/kusto/pom.mgmt.xml

```diff
@@ -13,5 +13,6 @@
     <module>mgmt-v2019_05_15</module>
     <module>mgmt-v2019_09_07</module>
     <module>mgmt-v2019_11_09</module>
+    <module>mgmt-v2020_02_15</module>
```

Comment:
I think this module could be added automatically. Is it due to you didn't add it after generation?

Reply:
But version and relativePath in pom.xml is correct. So I think Chuang is using the latest?

Reply:
Last PR didn't get merged due CI failure at that time. I just closed the pending one and use this new instead.